src/plugins/home/public/application/components/usecase_overview/search_use_case_setup.tsx

```diff
@@ -115,7 +117,7 @@ export const registerContentToSearchUseCasePage = (
     });
   });
 
-  const searchIcon = <EuiIcon color="subdued" size="l" type="search" />;
+  const searchIcon = <EuiIcon color="#9FC4D2" size="l" type="search" />;
```

Comment:
Shall we not hard code the color? Just wondering how it will look like under dark/light mode?

Reply:
dark mode
![image](https://github.com/user-attachments/assets/0f8ef6d7-7ac7-4b53-81ca-30af77a230f1)

light mode
![image](https://github.com/user-attachments/assets/946b8332-1403-4be4-9b44-d0ded704d40d)

Reply:
$ouiLinkColor Shall we use this variable?

https://oui.opensearch.org/1.14/#/guidelines/sass

Reply:
> $ouiLinkColor Shall we use this variable?
> 
> https://oui.opensearch.org/1.14/#/guidelines/sass

let me see if I can use predefined variables

Reply:
updated to a custom color by leveraging oui variable ` color: tint($euiLinkColor, 50%);`